packages/html/src/checkbox/templates/checkbox-label.tsx

```diff
@@ -2,7 +2,7 @@ import Checkbox from "../checkbox.spec";
 
 export const CheckboxLabel = ({ children, id, ...other }: any) => (
```

Comment:
This is unrelated to this PR, but can we please use the opportunity to rename the file/template to `CheckboxWithLabel`, as the current name applies it's the "Label of the Checkbox", but it's actually a "checkbox **and** a label"